### ISSUE TITLE
Add "list" and/or "detail" views to web ui

### ISSUE THREAD

**Issue by instinct-vfx** [arangodb/arangodb#5227]
## my environment running ArangoDB
I'm using the latest ArangoDB of the respective release series:
- [* ] 3.2

### this is a web interface-related issue:
I'm using the web interface with this browser:     running on this OS:

We have a growing number of collections, services and graphs and it has become increasingly confusing to browse them in the web ui. I think it would really help to have a list or detail view rather than just the icon view.  This could also display additional information that may be relevant (e.g. document count or the settings/info panes of collections etc.).

**Comment by 153957** [comment]
I completely agree with this. Working with >100 collections it is hard to find them. I always resort to 'Find...' in the browser to locate collections.

**Comment by baslr** [comment]
I observed myself scanning from left to right and top to bottom to find the collection in the collection view.
A list view could improve the UX.
